install_notifier.js

```diff
@@ -0,0 +1,7 @@
+// Dispatch a message to every URL that's in the manifest to say that RECAP is
+// installed.
+let currentVersion = chrome.runtime.getManifest().version;
+window.postMessage({
+  sender: "recap-extension",
+  message: currentVersion
```

Comment:
It seems unlikely this is the only message we will ever send. Since `window.postMessage()` automatically serializes objects, and indeed you're making use of that by sending an object with `sender` and `message` properties (rather than just a bare string), it seems like you should include a type, so that other messages can be disambiguated in the future.

I.e.:

```js
{
  sender: "recap-extension",
  type: "extension-version",
  message: currentVersion,
}
```
I suppose you could argue this is unnecessary since you've already added the nonstandard `sender` property which allows disambiguation, but I would still include more. Or rename `sender` to `type` and make its value `recap-extension-version`. Or something.

Reply:
I made this into:

```
    window.postMessage({
      sender: "recap-extension",
      message_name: "version",
      message: currentVersion
    }, "*");
```

So, `sender` lets us figure out what's sending the message (the RECAP extension), and then `message_name` gives us the variable name.